manim/animation/transform.py

```diff
@@ -357,21 +424,83 @@ def begin(self) -> None:
 
 
 class FadeToColor(ApplyMethod):
+    """Animation that changes color of a mobject.
+
+    Examples
+    --------
+
+    .. manim:: FadeToColor
+        :quality: low
+
+        class FadeToColor(Scene):
+            def construct(self):
+                self.play(FadeToColor(Text("Hello World!"), color=RED))
+
+    """
+
     def __init__(self, mobject: Mobject, color: str, **kwargs) -> None:
         super().__init__(mobject.set_color, color, **kwargs)
 
 
 class ScaleInPlace(ApplyMethod):
+    """Animation that scales a mobject by a certain factor.
+
+    Examples
+    --------
+
+    .. manim:: FadeToColor
+        :quality: low
+
+        class FadeToColor(Scene):
```

Comment:
Wrong animation class.

```suggestion
    .. manim:: ScaleInPlaceExample
        :quality: low

        class ScaleInPlaceExample(Scene):
```